pdns/recursordist/docs/changelog/4.5.rst

```diff
@@ -1,6 +1,126 @@
 Changelogs for 4.5.X
 ====================
 
+.. changelog::
+  :version: 4.5.0-alpha2
+  :released: 8th of March 2021
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 10010
+
+    Check sizeof(time_t) to be at least 8.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 10118
+
+    Change dnssec default to `process`.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 10047
+
+    Implement rfc 8198 - Aggressive Use of DNSSEC-Validated Cache.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 10112
+
+    Be less verbose telling we are looking up CNAMEs or DNAMEs while tracing.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 10111
+    :tickets: 10080
+
+    Handle policy (if needed) after postresolve and document the hooks better.
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9361
```

Comment:
I'm pretty sure that PR impacted only dnsdist?

Reply:
indeed